metering/metering-installing-metering.adoc

```diff
@@ -11,6 +11,37 @@ To get started installing metering, first install the Metering Operator from Ope
 
 include::modules/metering-install-operator.adoc[leveloffset=+1]
 
-include::modules/metering-install-metering-stack.adoc[leveloffset=+1]
+// Including this content directly in the assembly because the workflow requires linking off to the config docs, and we don't current link
+// inside of modules - klamenzo 2019-09-23
+[id="metering-install-metering-stack_{context}"]
+== Installing the Metering stack 
+
+To install the metering stack, first review the xref:../metering/configuring-metering/metering-about-configuring.adoc#metering-about-configuring[configuration options] and create a `MeteringConfig` resource. You can begin the process below to generate a default `MeteringConfig`, then use the examples in the documentation to modify this default file for your specific installation. 
+
+Review the following topics to create your `MeteringConfig`:
```

Comment:
Rework this section to be a list of prerequisites.

Reply:
Good call.